grails-app/services/au/org/ala/merit/OrganisationService.groovy

```diff
@@ -111,6 +114,28 @@ class OrganisationService {
         return result
     }
 
+    void regenerateReports(String id, List<String> organisationReportCategories = null, List<String> projectReportCategories = null) {
+        Map organisation = get(id)
+
+        regenerateOrganisationReports(organisation, organisationReportCategories)
+    }
+
+    private void regenerateOrganisationReports(Map organisation, List<String> reportCategories = null) {
+
+        List organisationReportConfig = organisation.config?.organisationReports
+        ReportOwner owner = new ReportOwner(
+            id:[organisationId:organisation.organisationId],
+            name:organisation.name,
+            periodStart:organisation.startDate,
```

Comment:
periodStart/periodEnd don't make sense for an organisation.  I am writing a change to move these into config.